docker-compose.yml

```diff
@@ -27,6 +27,7 @@ services:
             VIRTUAL_HOST: "${VIRTUAL_HOST}"
             LETSENCRYPT_HOST: "${LETSENCRYPT_HOST}"
             LETSENCRYPT_EMAIL: "${LETSENCRYPT_EMAIL}"
-            SLACK_SIGNING_SECRET: "${SLACK_SIGNING_SECRET}"
+            SLACK_SIGNING_SECRET: "${SLACK_SIGNING_SECRET}
```

Comment:
Missing `"`! I'll go ahead and push that as an update so this can get merged right away.